### DISCUSSION TITLE
Images won't display in Edge

### DISCUSSION THREAD

**Discussion by github-account1111** [gildas-lormeau/SingleFile#1384] [Q&A]
For questions, please create or update a thread here: 
https://github.com/gildas-lormeau/SingleFile/discussions

Please ensure that you do not find an answer before reporting the issue:
 - in the FAQ: https://github.com/gildas-lormeau/SingleFile/blob/master/faq.md
 - in the list of known issues: https://github.com/gildas-lormeau/SingleFile#known-issues
 - in the help page embedded in the extension

**Describe the bug**
Images aren't saved for https://www.listingsproject.com/listings/industrial-koreatown-studio-loft-cef66d95-67c9-4dfc-947d-e0ade464a90b and other listings on the site.

**To Reproduce**
1. Go to https://www.listingsproject.com/listings/industrial-koreatown-studio-loft-cef66d95-67c9-4dfc-947d-e0ade464a90b
1. Save the page
1. Open the file

**Expected behavior**
Faithful copy, images are shown.

**Screenshots**

![image](https://github.com/gildas-lormeau/SingleFile/assets/5728319/e02a98fc-70ce-489c-ace3-105112dd2e95)

![image](https://github.com/gildas-lormeau/SingleFile/assets/5728319/f7e1b7a1-a012-4c84-a203-71c5cfa48337)

![image](https://github.com/gildas-lormeau/SingleFile/assets/5728319/1f700407-b217-4c0d-bc61-d0c668ed478c)

![image](https://github.com/gildas-lormeau/SingleFile/assets/5728319/7941482e-76f7-41c1-aa61-d1d6e873220c)

**Environment**
 - OS: Win10
 - Browser: Edge
 - Version: 120.0.2210.77

**Additional context**
I've tried deferred images on and off. I've also reset all the settings, to no avail.

**Comment by gildas-lormeau** [comment]
In fact, images are saved in the [AVIF format](https://en.wikipedia.org/wiki/AVIF) but Edge cannot display them. To circumvent this limitation, you can set the value of the option `Network > "Accept" headers > videos` to `image/webp,image/apng,image/svg+xml,image/*,*/*;q=0.8`.
